doc/installation_osx.md

```diff
@@ -6,21 +6,21 @@
 
 ## Building from Source
 
-1. Install CommantLineTools `sudo xcode-select` or download XCode 6.0+ via the AppStore
+1. Install CommantLineTools `sudo xcode-select --install` or download XCode 6.0+ via the AppStore
 2. Install the Homebrew package manager via terminal - [link](http://brew.sh/)
 3. Install the following packages via brew:
   * `brew install cmake libusb pkg-config`
-  * `brew cask install vulkan-sdk`
+  * `brew cask install apenngrace/vulkan/vulkan-sdk`
```

Comment:
@dorodnic, can we just remove this line?  Or move it to it's own "Optional" section?  The `vulkan-sdk` is not required.  Maybe we should suggest `brew install vulkan-headers` instead?

Reply:
It wasn't originally in the guide (and to my understanding not supposed to be), but people reported it helped them with GLFW compilation